### DISCUSSION TITLE
(Solved)How Should I Enable WakaTime Plugin?

### DISCUSSION THREAD

**Discussion by bydmiller** [NotAShelf/nvf#293] [General]
I tried to use `:WakaTimeApiKey` but the command is not recognized. I have vim-wakatime plugin enabled.

**Comment by NotAShelf** [comment]
Enabling `vim-wakatime` and running `:WakaTimeApiKey` is exactly how the setup goes.

Looks like it works on the default configuration package, so I'll need a minimal reproduction example (assuming this is a plugin conflict).

**Comment by bydmiller** [comment]
I am not sure what was the issue. But it seems to be working for now. Thank you.
